include/SFML/Graphics/Texture.hpp

```diff
@@ -52,6 +53,9 @@ class SFML_GRAPHICS_API Texture : GlResource
     ////////////////////////////////////////////////////////////
     /// \brief Types of texture coordinates that can be used for rendering
     ///
+    /// This enum is present for backward compatibility only. Prefer using
+    /// sf::CoordinateType instead.
+    ///
```

Comment:
What about changing this `enum` to a `typedef` referring to `sf::CoordinateType`?
Then we don't have to deprecate `bind()`, just this typedef (not sure if possible with `SFML_DEPRECATED`).

Reply:
That is what I did at first, and discovered that indeed the deprecation propagates to the typedef too :(

Reply:
I would still do it, we can also just mention "deprecated" in the docs.
There was no standard deprecation mechanism in C++ until recently, so we have to live with the options available...

Reply:
I've re-targeted this PR against `master` now, so I just removed the old function since we don't need to preserve backward compatibility in 3.0.